src/main.ts

```diff
@@ -62,7 +62,7 @@ async function runForReleaseEvent(vt: VirusTotal) {
   core.info(`Release event detected for ${github.context().ref} in this workflow. Preparing to scan assets...`);
 
   const release = await github.getRelease(octokit, github.context().ref.replace('refs/tags/', ''));
-  release.body = release.body.concat(`\n\n🛡 [VirusTotal GitHub Action](https://github.com/crazy-max/ghaction-virustotal) analysis:`);
```

Comment:
Let's just keep the link as it was before.

Reply:
Not possible as written in the git commit: The collapsible summary text is clickable to expand it and thus links won't
be rendered as such ... Thats why adding it at the end.

Reply:
Ah yes indeed didn't catch that